### DISCUSSION TITLE
Item Dropped Timer

### DISCUSSION THREAD

**Discussion by Full-Zip** [runelite/runelite#16186] [Ideas]
I'm wondering if someone can make plugin, that does. When a person drops a item like "food"/ anything on the ground. It will take 60 seconds until it appears for everyone around you. Can someone please make a Plugin, that adds a timer above that item, that is dropped on the ground. From 60 seconds to count down to 1. I find this could help a lot of people at end game content.

**Comment by Enriath** [comment]
`Ground Items` can already show item despawn timers, and when the timer changes from green to yellow, the item is visible to others.

**Reply by Full-Zip** [reply]
I wouldn't be able to pick it up in time, before it drops if needed.

**Comment by Full-Zip** [comment]
Boss with this timer with sharing food and potions would be helpful. so they know what to do before boss spawns.

**Reply by Enriath** [reply]
You're going to have to be more specific about what you mean here

**Comment by Full-Zip** [comment]
I found the plug in you were talking about. It was hard to find. But it does have a timer for ground dropped items.
